dev/ci/gitlab.bat

```diff
@@ -39,6 +39,11 @@ SET PATH=%PATH%;C:\Program Files\7-Zip\;C:\Program Files\Microsoft SDKs\Windows\
 
 IF "%WINDOWS%" == "enabled_all_addons" (
   SET EXTRA_ADDONS=^
+    -addon=aactactics ^

```

Comment:
Sneakily start building aac in all_addons mode?

Reply:
Right. It was commented out because it was not working with 8.9 and master but it should now. Doing this change as part of this PR might not be a good idea though because this change should also be backported.